Apt.py

```diff
@@ -0,0 +1,66 @@
+"""Apt extension
+
+Note that it is up to you to ensure that the package lists are up to date"""
+
+import os
+import subprocess
+from shutil import which
+from albertv0 import *
+import re
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Apt"
+__version__ = "1.0"
+__trigger__ = "apt"
+__author__ = "Harindu Dilshan"
+__dependencies__ = ['apt','gksu','pkexec']
+
+if which("apt") is None:
```

Comment:
maybe using apt-get is better since apt is a userfriendly frontend that may break backward compatibility